### ISSUE TITLE
show image from description if exists

### ISSUE THREAD

**Issue by eileenmcnaughton** [mlutfy/civicrmogp#3]
I'm working on a client feature request to add the 'first image from the description field if exists' with the fall back position being the site logo. Will do a PR if you are happy to accept this feature

**Comment by mlutfy** [comment]
Hmm, I agree it's a good feature, but I kind of wish we had something more structured. Might also cause unexpected changes in upgrades. Mind leaving it in a fork/sub-module and we can see later?

**Comment by eileenmcnaughton** [comment]
sure - I offered the more complex option of storing to a custom field or something but client wants this simple approach at this stage

**Comment by mlutfy** [comment]
On the main client site that uses this module, we have a big banner image stored in a specific directory (ex: /files/banners/contribute-123.jpg, where 123 is the contribution form ID). Makes it easier for theming, and less risk when editing the description in the wysiwyg. If we could have something equivalent for the ogp image, would be great, but presumably your client wants a more visual way of handling it?

**Comment by eileenmcnaughton** [comment]
yeah - I guess they have many many contribution pages created by many many people

**Comment by eileenmcnaughton** [comment]
OK - this is what I'm using to crawl this image out of the text / fall back to logo 

https://github.com/eileenmcnaughton/civicrmogp/commit/7f113b2e74caadf9176564189c649fc25fe5c0de
